subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -791,17 +811,44 @@ void ticker_worker(void *param)
 		 */
 		if (ticker->ticks_slot != 0U &&
 		   (slot_reserved || ticker_resolve_collision(node, ticker))) {
+#if defined(CONFIG_BT_TICKER_EXT)
+			if (ticker->ext_data &&
+			    ticker->ext_data->ticks_slot_window != 0U &&
+			    ticker->ext_data->reschedule_state ==
+			    TICKER_RESCHEDULE_STATE_NONE &&
+			   (ticker->lazy_periodic <= ticker->lazy_current)) {
+				/* Mark node for re-scheduling in ticker_job */
+				ticker->ext_data->reschedule_state =
+					TICKER_RESCHEDULE_STATE_PENDING;
+			} else if (ticker->ext_data) {
+				/* Mark node as not re-scheduling */
+				ticker->ext_data->reschedule_state =
+					TICKER_RESCHEDULE_STATE_NONE;
+			}
+#endif /* CONFIG_BT_TICKER_EXT */
```

Comment:
Here a collision has been detected. It is checked if node has slot_window set, and if so set the reschedule state pending for ticker_job.